### ISSUE TITLE
Roles GUI css and js implantation is outside of html 

### ISSUE THREAD

**Issue by stars8422** [jeremykenedy/laravel-roles#72]
When i enable roles gui then see my source code - i see  bootstrap file import, css class, js file import and js script are outside of html  code.
Code Like this :

  | <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css">
-- | --
  | <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  | <style type="text/css" media="screen">
  | .no-padding {
  | padding: 0;
  | }
  | .pointer {
  | cursor: pointer;
  | }
  | .card {
  | width: 100%;
  | }
  | .modal .modal-header {
  | -webkit-border-top-left-radius: .2rem;
  | -webkit-border-top-right-radius: .2rem;
  | -moz-border-radius-topleft: .2rem;
  | -moz-border-radius-topright: .2rem;
  | border-top-left-radius: .2rem;
  | border-top-right-radius: .2rem;
  | }
  | .modal-success .modal-header {
  | color: #ffffff;
  | background-color: #5cb85c;
  | }
  | .modal-warning .modal-header {
  | color:#fff;
  | background-color: #f0ad4e;
  | }
  | .modal-danger .modal-header {
  | color:#fff;
  | background-color: #d9534f;
  | }
  | .modal-info .modal-header {
  | color:#fff;
  | background-color: #5bc0de;
  | }
  | .modal-primary .modal-header {
  | color:#fff;
  | background-color: #428bca;
  | }
  | .blocked-table {
  | border: 0;
  | }
  | .blocked-table tr td:first-child {
  | padding-left: 15px;
  | }
  | .blocked-table tr td:last-child {
  | padding-right: 15px;
  | }
  | .blocked-table.table-responsive,
  | .blocked-table.table-responsive table {
  | margin-bottom: 0;
  | }
  | .clickable-row:hover {
  | cursor: pointer;
  | }
  | .blocked-table.table-responsive {
  | border: none;
  | }
  | .blocked-table .list-group {
  | margin-bottom: 0;
  | }
  | .blocked-table .dropdown-menu > li button {
  | padding: 3px 20px;
  | background: transparent;
  | border: none;
  | outline: none;
  | width: 100%;
  | text-align: left;
  | white-space: nowrap;
  | }
  | .blocked-table .dropdown-menu > li button:hover {
  | background: rgba(0,0,0,.04);
  | }
  |  
  | .blocked-table {
  | border: 0;
  | }
  | .blocked-table tr th {
  | border: 0 !important;
  | }
  | .blocked-table tr th:first-child,
  | .blocked-table tr td:first-child {
  | padding-left: 15px;
  | }
  | .blocked-table tr th:last-child,
  | .blocked-table tr td:last-child {
  | padding-right: 15px;
  | }
  | .blocked-table .table-responsive,
  | .blocked-table .table-responsive table {
  | margin-bottom: 0;
  | border-top: 0;
  | border-left: 0;
  | border-right: 0;
  | }
  | .blocked-table .data-table {
  | width: 100%;
  | }
  | .btn-save,
  | .pw-change-container {
  | display: none;
  | }
  | .has-error,
  | .has-error input,
  | .has-error select,
  | .has-error textarea {
  | border-color: #d9534f;
  | }
  | .clear-search {
  | display: none;
  | }
  | div.dataTables_wrapper div.dataTables_length label {
  | margin-top: 5px;
  | margin-bottom: 5px;
  | margin-right:20px;
  | }
  | div.dataTables_wrapper div.dataTables_length label select {
  | margin-left: 5px;
  | margin-right: 5px
  | }
  | #search_blocked_form {
  | display: none;
  | }
  | .disabled {
  | color: #dcdbdb;
  | pointer-events: none;
  | cursor: not-allowed;
  | }
  | .disabled .switch span.active {
  | color: #dcdbdb;
  | pointer-events: none;
  | cursor: not-allowed;
  | }
  | .accordion.accordion-item {
  | cursor: pointer;
  | cursor: zoom-out;
  | }
  | .accordion.accordion-item.collapsed {
  | cursor: zoom-in;
  | }
  | .accordion.accordion-item .permission-name:before,
  | .accordion.accordion-item .role-name:before {
  | font-family: 'FontAwesome';
  | content: "\f068";
  | margin: 0 .5em 0 0;
  | }
  | .accordion.accordion-item.collapsed .permission-name:before,
  | .accordion.accordion-item.collapsed .role-name:before {
  | content: "\f067";
  | }
  | .list-group-item table {
  | font-size: .5em;
  | width: 100%;
  | display: table;
  | table-layout: fixed;
  | }
  | .permissions-table,
  | .roles-table {
  | font-size: .9em;
  | }
  | .table.permissions-table thead th,
  | .table.permissions-table td,
  | .table.roles-table thead th,
  | .table.roles-table td {
  | border: none;
  | vertical-align: middle;
  | }
  | .table.permissions-table form,
  | .table.roles-table form {
  | margin: 0;
  | }
  | @media(min-width: 576px) {
  | .edit-form-delete {
  | margin-top: -2.85em;
  | }
  | }
  | </style>
  | <style type="text/css" media="screen">
  | .visible-xs,
  | .visible-sm,
  | .visible-md,
  | .visible-lg {
  | display: none !important;
  | }
  | .visible-xs-block,
  | .visible-xs-inline,
  | .visible-xs-inline-block,
  | .visible-sm-block,
  | .visible-sm-inline,
  | .visible-sm-inline-block,
  | .visible-md-block,
  | .visible-md-inline,
  | .visible-md-inline-block,
  | .visible-lg-block,
  | .visible-lg-inline,
  | .visible-lg-inline-block {
  | display: none !important;
  | }
  | @media (max-width: 767px) {
  | .visible-xs {
  | display: block !important;
  | }
  | table.visible-xs {
  | display: table !important;
  | }
  | tr.visible-xs {
  | display: table-row !important;
  | }
  | th.visible-xs,
  | td.visible-xs {
  | display: table-cell !important;
  | }
  | }
  | @media (max-width: 767px) {
  | .visible-xs-block {
  | display: block !important;
  | }
  | }
  | @media (max-width: 767px) {
  | .visible-xs-inline {
  | display: inline !important;
  | }
  | }
  | @media (max-width: 767px) {
  | .visible-xs-inline-block {
  | display: inline-block !important;
  | }
  | }
  | @media (min-width: 768px) and (max-width: 991px) {
  | .visible-sm {
  | display: block !important;
  | }
  | table.visible-sm {
  | display: table !important;
  | }
  | tr.visible-sm {
  | display: table-row !important;
  | }
  | th.visible-sm,
  | td.visible-sm {
  | display: table-cell !important;
  | }
  | }
  | @media (min-width: 768px) and (max-width: 991px) {
  | .visible-sm-block {
  | display: block !important;
  | }
  | }
  | @media (min-width: 768px) and (max-width: 991px) {
  | .visible-sm-inline {
  | display: inline !important;
  | }
  | }
  | @media (min-width: 768px) and (max-width: 991px) {
  | .visible-sm-inline-block {
  | display: inline-block !important;
  | }
  | }
  | @media (min-width: 992px) and (max-width: 1199px) {
  | .visible-md {
  | display: block !important;
  | }
  | table.visible-md {
  | display: table !important;
  | }
  | tr.visible-md {
  | display: table-row !important;
  | }
  | th.visible-md,
  | td.visible-md {
  | display: table-cell !important;
  | }
  | }
  | @media (min-width: 992px) and (max-width: 1199px) {
  | .visible-md-block {
  | display: block !important;
  | }
  | }
  | @media (min-width: 992px) and (max-width: 1199px) {
  | .visible-md-inline {
  | display: inline !important;
  | }
  | }
  | @media (min-width: 992px) and (max-width: 1199px) {
  | .visible-md-inline-block {
  | display: inline-block !important;
  | }
  | }
  | @media (min-width: 1200px) {
  | .visible-lg {
  | display: block !important;
  | }
  | table.visible-lg {
  | display: table !important;
  | }
  | tr.visible-lg {
  | display: table-row !important;
  | }
  | th.visible-lg,
  | td.visible-lg {
  | display: table-cell !important;
  | }
  | }
  | @media (min-width: 1200px) {
  | .visible-lg-block {
  | display: block !important;
  | }
  | }
  | @media (min-width: 1200px) {
  | .visible-lg-inline {
  | display: inline !important;
  | }
  | }
  | @media (min-width: 1200px) {
  | .visible-lg-inline-block {
  | display: inline-block !important;
  | }
  | }
  | @media (max-width: 767px) {
  | .hidden-xs {
  | display: none !important;
  | }
  | }
  | @media (min-width: 768px) and (max-width: 991px) {
  | .hidden-sm {
  | display: none !important;
  | }
  | }
  | @media (min-width: 992px) and (max-width: 1199px) {
  | .hidden-md {
  | display: none !important;
  | }
  | }
  | @media (min-width: 1200px) {
  | .hidden-lg {
  | display: none !important;
  | }
  | }
  | .visible-print {
  | display: none !important;
  | }
  | @media  print {
  | .visible-print {
  | display: block !important;
  | }
  | table.visible-print {
  | display: table !important;
  | }
  | tr.visible-print {
  | display: table-row !important;
  | }
  | th.visible-print,
  | td.visible-print {
  | display: table-cell !important;
  | }
  | }
  | .visible-print-block {
  | display: none !important;
  | }
  | @media  print {
  | .visible-print-block {
  | display: block !important;
  | }
  | }
  | .visible-print-inline {
  | display: none !important;
  | }
  | @media  print {
  | .visible-print-inline {
  | display: inline !important;
  | }
  | }
  | .visible-print-inline-block {
  | display: none !important;
  | }
  | @media  print {
  | .visible-print-inline-block {
  | display: inline-block !important;
  | }
  | }
  | @media  print {
  | .hidden-print {
  | display: none !important;
  | }
  | }
  | </style>
  | <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  | <script type="text/javascript">
  | $(function() {
  | // Confirm Form Submit Modal
  | $('#confirmDelete').on('show.bs.modal', function (e) {
  | var message = $(e.relatedTarget).attr('data-message');
  | var title = $(e.relatedTarget).attr('data-title');
  | var form = $(e.relatedTarget).closest('form');
  | $(this).find('.modal-body p').text(message);
  | $(this).find('.modal-title').text(title);
  | $(this).find('.modal-footer #confirm').data('form', form);
  | });
  | $('#confirmDelete').find('.modal-footer #confirm').on('click', function(){
  | $(this).data('form').submit();
  | });
  | });
  | </script>
  | <script type="text/javascript" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
  | <script type="text/javascript" src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>
  | <script type="text/javascript">
  | $(document).ready(function() {
  | $('.data-table').dataTable({
  | "order": [[0]],
  | "pageLength": 100,
  | "lengthMenu": [
  | [10, 25, 50, 100, 500, 1000, -1],
  | [10, 25, 50, 100, 500, 1000, "All"]
  | ],            "paging": true,
  | "lengthChange": true,
  | "searching": true,
  | "ordering": true,
  | "info": true,
  | "autoWidth": false,
  | "dom": 'T<"clear">lfrtip',
  | "sPaginationType": "full_numbers",
  | 'aoColumnDefs': [{
  | 'bSortable': false,
  | 'searchable': false,
  | 'aTargets': ['no-search'],
  | 'bTargets': ['no-sort']
  | }]
  | });
  | });
  | </script>
  | <script type="text/javascript">
  | $(document).ready(function(){
  | var is_touch_device = 'ontouchstart' in document.documentElement;
  | if (!is_touch_device) {
  | $('[data-toggle="tooltip"]').tooltip();
  | }
  | });
  | </script>
  |  
  | <!DOCTYPE html>
  | <html lang="en">
  | <head>
  | <meta charset="utf-8">
  | <meta name="viewport" content="width=device-width, initial-scale=1">
  |  
  | 
  | <meta name="csrf-token" content="hwLzm5GLj4iaoeXKtVHtD8siDFSqifFqkK2gfyco">
  |  
  | <title>Laravel</title>
  | <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
  | <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  | </head>
  | <body>

**Comment by jeremykenedy** [comment]
Something is wrong with your app.

https://github.com/jeremykenedy/laravel-roles#opening-an-issue
